content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,211 @@
+
+2. Add two files named send-xrp.js and send-xrp.html. You can also create a separate css file depending on your choice. 
+
+3. Copy the contents of the [send-xrp.html](_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.html) file to your send-xrp.html file. The provided HTML code includes three input fields for the destination address, amount, and destination tag, each with their corresponding labels.
+
+4. Now that we have the HTML code, let's add the JavaScript code. In helpers folder, add a new file named submit-transaction.js and copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/helpers/submit-transaction.js", language="js") }}
+
+In the code above, we are using the [submit](https://xrpl.org/submit.html) method to submit the signed transaction to the XRPL. You can read more about the submit method [here](https://xrpl.org/submit.html).
+
+5. Now back to the send-xrp.js file, copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.js", language="js") }}
+
+In the code above, we retrieve elements from DOM, add event listeners, validate inputs and after validation, we call the `submitTransaction` function from the submit-transaction.js file to submit the transaction to the XRPL.
+
+### Step 4: Creating the Transactions Page
+
+Now that we have created the home page and the send XRP page, let's create the transactions page that will display the transaction history of the account.
+
+  1. Account: The account that sent the transaction.
+  2. Destination: The account that received the transaction.
+  3. Amount: The amount of XRP sent in the transaction.
+  4. Transaction Type: The type of transaction.
+  5. Result: The result of the transaction.
+  6. Link: A link to the transaction on the XRP Ledger Explorer.
+
+![Transactions Page Screenshot](img/js-wallet-transaction.png)
+
+1. Create a folder named transaction-history in the src directory.
+
+2. Add two files named transaction-history.js and transaction-history.html.
+
+3. In the transaction-history.js file, copy the code written below.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.js", language="js") }}
+
+In the code above we fetch the transactions from the XRP Ledger by using the [account_tx](https://xrpl.org/account_tx.html) method. You can read more about the account_tx method [here](https://xrpl.org/account_tx.html).
+
+4. Copy the contents of the [transaction-history.html](_code-samples/build-a-wallet/js/src/transaction-history/transaction-history.html) file to your transaction-history.html file.
+
+You can use this code as a starting point and customize it to suit your needs. For example, you might want to add more columns to the table. You can also check out the [XRP Ledger Explorer](https://livenet.xrpl.org/) to see how the transaction details are displayed.
+
+### Step 5: Running the App
```

Comment:
I think we should do this earlier - it's better if the user is able to see the results of each step after they do it, and so having them do the necessary setup near the beginning let's them feel proud of each step, rather than waiting until the very end to play with what they're building.

Reply:
It's Step 2 now